tools/utils/updatePackageReferences.js

```diff
@@ -3,7 +3,7 @@
  * Updates devDependencies with paste peerDependencies
  */
 const chalk = require('chalk');
-const lodash = require('lodash');
+import difference from 'lodash/difference';
```

Comment:
question: can we use `import` syntax in this JS file? `node tools/utils/updatePackageReference.js` in terminal works?

Reply:
tested this and it does not work. Changing this back to `require`, along w/ other uses of lodash in `tools`